pkg/api/v1beta3/types.go

```diff
@@ -1609,3 +1610,19 @@ type SecretList struct {
 
 	Items []Secret `json:"items" description:"items is a list of secret objects"`
 }
+
+// ServerStatus (and ServerStatusList) holds the cluster validation info.
+type ServerStatus struct {
+	Component  string       `json:"component,omitempty"`
+	Health     string       `json:"health,omitempty"`
```

Comment:
What is this field populated with? All fields must bear description tags, which will show up in swagger.